staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -25,6 +25,18 @@ import (
 	"k8s.io/apiserver/pkg/authentication/user"
 )
 
+// Some commonly useful constituents
+var (
+	verbAll       = []string{flowcontrol.VerbAll}
+	apiGroupAll   = []string{flowcontrol.APIGroupAll}
+	resourceAll   = []string{flowcontrol.ResourceAll}
+	nsEvery       = []string{"*"}
+	ruleRscAll    = []flowcontrol.ResourcePolicyRule{resourceRule(verbAll, apiGroupAll, resourceAll, nsEvery, true)}
```

Comment:
Nit: Personally, I'd spell it out, since this is file-wide-- probably everyone knows what "ns" is but "Rsc" isn't obviously "Resource".

(next line too)

Reply:
done